app/models/enterprise.rb

```diff
@@ -100,9 +100,11 @@ class Enterprise < ApplicationRecord
   validate :shopfront_taxons
   validate :shopfront_producers
   validate :enforce_ownership_limit, if: lambda { owner_id_changed? && !owner_id.nil? }
+  validates :phone, phone: { allow_blank: true, country_specifier: ->(instance) { instance.address.country.iso } }, unless: lambda { address.nil? }
```

Comment:
Question for @mkllnk: can one instance sometimes host several countries?

Reply:
Yes, that's possible, as far as I know.